openreview/conference/builder.py

```diff
@@ -1540,6 +1540,30 @@ def is_release_authors(is_note_accepted):
             self.set_homepage_decisions(decision_heading_map=decision_heading_map)
         self.client.remove_members_from_group('active_venues', self.id)
 
+    def send_decision_notifications(self, decision, message):
+        decision_notes = self.client.get_all_notes(
+            invitation=self.get_invitation_id(self.decision_stage.name, '.*'),
+            content={
+                'decision': decision
+            }
+        )
+        paper_notes = {n.forum: n for n in self.get_submissions()}
+
+        def send_notification(note):
+            paper_note = paper_notes[note.forum]
+            subject = "[{SHORT_NAME}] Decision notification for your submission {submission_number}: {submission_title}".format(
+                SHORT_NAME=self.get_short_name(),
+                submission_number=paper_note.number,
+                submission_title=paper_note.content['title']
+            )
+            final_message = f'''{message.format(
+                submission_title=paper_note.content['title'],
```

Comment:
what happens if the PC removes any of these token? do we get an error, right? how can we avoid it?

I think `submission_number` and `forum_url` should be in the email content too.

Reply:
here is an example from NeurIPS:

```
f'''Dear {{{{firstname}}}},

We are delighted to inform you that your submission {submission.number}: {submission.content['title']} has been accepted for a poster presentation at NeurIPS 2021. Congratulations!

There were 9122 full paper submissions to NeurIPS this year, of which the program committee accepted 26% for presentation at the conference.  You can find the final reviews for your paper on the submission’s page in OpenReview:

https://openreview.net/forum?id={submission.id}

Please take the reviews and the meta-review of your paper into careful consideration.  Your response may have played an important role in the decisions reached by the program committee, so please make sure that you follow up on all the revisions promised in your response to reviews.  Since your paper was accepted, the reviews for your paper and any discussion visible to you and the reviewers assigned to your paper (but not any confidential exchanges you may have had with the Area Chair or the Senior Area Chair) will all become public.

This year, all accepted papers will be presented both in a pre-recorded talk (available on-demand) and at one 90-minute virtual poster session.  The deadline for uploading your video recording is October 18, 2021.  The final camera-ready version of your paper is due on October 26, 2021.  Further instructions regarding both will be sent by email shortly.  We will also be asking for your poster session time preferences.

Meanwhile, we ask that you make sure that the set of keywords you specified for your paper in OpenReview accurately capture the essence of this work.

Congratulations again, and we look forward to seeing you virtually in December!  Registration information is already available on the conference webpage.

Alina Beygelzimer, Yann Dauphin, Percy Liang, and Jenn Wortman Vaughan
NeurIPS 2021 Program Chairs

'''
```